prototype/p2p/pre_p2p/module.go

```diff
@@ -126,8 +129,8 @@ func (m *networkModule) BroadcastMessage(msg *anypb.Any, topic string) error {
 }
 
 func (m *networkModule) Send(addr string, msg *anypb.Any, topic string) error {
-	c := &types.P2PMessage{
-		Topic: topic, // TODO topic is either P2P (from this module) or consensus
+	c := &p2ptypes.NetworkMessage{
+		Topic: p2ptypes.Topic(topic), // TODO topic is either P2P (from this module) or consensus
```

Comment:
Add names for TODOs. Could be someone else or just "team" if it can be handled by anyone.

Reply:
Those were yours. Should I olshansky'em?